app/views/spotlight/appearances/edit.html.erb

```diff
@@ -32,22 +33,23 @@
           <% end %>
           <%= field_set_tag(t(:'.site_masthead.source.header')) do %>
             <div>
-              <%= m.radio_button(:source, :exhibit, disabled: 'disabled', label: t(:'.site_masthead.source.exhibit.label')) %>
+              <%= m.radio_button(:source, :exhibit, label: t(:'.site_masthead.source.exhibit.label')) %>
               <div>
-                <%= m.text_field_without_bootstrap(:document_global_id, class: "form-control", disabled: 'disabled') %>
+                <%= m.hidden_field(:document_global_id) %>
+                <%= m.hidden_field(:remote_image_url, data: jcrop_options) %>
+                <%= text_field_tag(:document_title, presenter(m.object.document || SolrDocument.new).document_show_html_title, class: "form-control", data: {input_select_target: '#appearance_masthead_source_exhibit', masthead_typeahead: true, target_panel: '#masthead-target-panel', :'id-field' => '#appearance_masthead_document_global_id', remote_url_field: '#appearance_masthead_remote_image_url' }) %>
```

Comment:
Don't hard-code `SolrDocument` here. Pull it out of the exhibit config instead (`current_exhibit.blacklight_configuration.solr_document_model` in Blacklight 5.9). Maybe you could get away without needed it, even.